### PR TITLE
Add python-pkcs11 to conda-forge

### PR DESCRIPTION
**Pull request by sodre**

### CODE REVIEW

**Comment by sodre** [comment]
@ocefpaf,
can you take a quick look at why the Windows build is failing  even though both recipes have "skip: true  # [win]" ?

File: recipes/python-pkcs11/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - pkcs11
+
+about:
+  home: http://python-pkcs11.readthedocs.io/
+  license: MIT
+  license_family: MIT
+  summary: 'A high level Python interface to the PKCS#11 (Cryptoki) standard to support HSM and Smartcard devices'
+
+  description: |
+      A high level, “more Pythonic” interface to the PKCS#11 (Cryptoki) standard to support
```
Comment (ocefpaf) [inline recipes/python-pkcs11/meta.yaml]: @sodre the Windows issue is probably with these unicode quotes here.

**Comment by ocefpaf** [comment]
@sodre Windows is failing to parse the recipe, not to build. Conda could not reach the skip statement you have there. Issues like these are usually some bad character in the recipe that Python 2.7 on Windows cannot handle.

**Comment by sodre** [comment]
@ocefpaf  and @CJ-Wright, this is now ready for review. 

sorry it took so long...

File: recipes/aenum/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (CJ-Wright) [inline recipes/aenum/meta.yaml]: Can this be noarch?

Comment (sodre) [reply recipes/aenum/meta.yaml]: It does not look like it does. The setup.py code has a py2 vs py3 check.

**Comment by sodre** [comment]
Do you need any other changes before it is merged?

File: recipes/python-pkcs11/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    - toolchain
+  run:
+    - cython
+    - aenum 2.0.*
+    - asn1crypto ==0.22.0
+    - cached-property 1.3.*
+
+test:
+  imports:
+    - pkcs11
```
Comment (CJ-Wright) [inline recipes/python-pkcs11/meta.yaml]: Can we get some more import tests here?

Comment (sodre) [reply recipes/python-pkcs11/meta.yaml]: your wish is my command

**Comment by CJ-Wright** [comment]
LGTM

**Pull request merged**